pkg/webhook/webhook.go

```diff
@@ -48,11 +58,41 @@ const (
 	FleetWebhookCfgName      = "fleet-validating-webhook-configuration"
 	FleetWebhookSvcName      = "fleetwebhook"
 
-	crdResourceName           = "customresourcedefinitions"
-	memberClusterResourceName = "memberclusters"
-	namespaceResouceName      = "namespaces"
-	replicaSetResourceName    = "replicasets"
-	podResourceName           = "pods"
+	crdResourceName                      = "customresourcedefinitions"
+	bindingResourceName                  = "bindings"
+	configMapResourceName                = "configmaps"
+	endPointResourceName                 = "endpoints"
+	eventResourceName                    = "events"
+	limitRangeResourceName               = "limitranges"
+	persistentVolumeClaimsName           = "persistentvolumeclaims"
+	podTemplateResourceName              = "podtemplates"
+	replicationControllerResourceName    = "replicationcontrollers"
+	resourceQuotaResourceName            = "resourcequotas"
+	secretResourceName                   = "secrets"
+	serviceAccountResourceName           = "serviceaccounts"
+	servicesResourceName                 = "services"
+	controllerRevisionResourceName       = "controllerrevisions"
+	daemonSetResourceName                = "daemonsets"
+	deploymentResourceName               = "deployments"
+	statefulSetResourceName              = "statefulsets"
+	localSubjectAccessReviewResourceName = "localsubjectaccessreviews"
+	horizontalPodAutoScalerResourceName  = "horizontalpodautoscalers"
+	cronJobResourceName                  = "cronjobs"
+	jobResourceName                      = "jobs"
+	workResourceName                     = "works"
+	leaseResourceName                    = "leases"
+	endPointSlicesResourceName           = "endpointslices"
+	ingressResourceName                  = "ingresses"
+	networkPolicyResourceName            = "networkpolicies"
+	podDisruptionBudgetsResourceName     = "poddisruptionbudgets"
+	roleResourceName                     = "roles"
+	roleBindingResourceName              = "rolebindings"
+	csiStorageCapacityResourceName       = "csistoragecapacities"
+	memberClusterResourceName            = "memberclusters"
+	internalMemberClusterResourceName    = "internalmemberclusters"
+	namespaceResourceName                = "namespaces"
+	replicaSetResourceName               = "replicasets"
+	podResourceName                      = "pods"
```

Comment:
IIRC, we have a pod webhook already.

Reply:
We have validating webhook for pod, replicaset which block their creation is fleet-, kube- prefixed namespaces. I guess we can remove these validating webhook configurations since we have fleet guard rail,

- https://github.com/Azure/fleet/blob/main/pkg/webhook/pod/pod_validating_webhook.go
- https://github.com/Azure/fleet/blob/main/pkg/webhook/replicaset/replicaset_validating_webhook.go